package.json

```diff
@@ -4,9 +4,7 @@
   "description": "",
   "dependencies": {
     "@aragon/os": "^4.2.1",
-    "@aragon/apps-shared-minime": "^1.0.1",
-    "@aragon/apps-token-manager": "2.1.0",
-    "@aragon/apps-voting": "2.1.0"
```

Comment:
We're using the Agent, Finance, Token Manager, Vault, and Voting apps in the bundled template, so we should also include them in the `devDependencies` (putting them as production dependencies before was a mistake).